static/js/stream_data.js

```diff
@@ -383,7 +383,7 @@ export function get_subscribed_streams_for_user(user_id) {
     // Note that we only have access to subscribers of some streams
     // depending on our role.
     const all_subs = get_unsorted_subs();
-    const subscribed_subs = [];
+    const subscribed_streams = [];
```

Comment:
For this one, I think renaming the conflicting function to `subscribed_stream_names` might be better for consistency with the existing `subscribed_stream_ids`.